### PR TITLE
Mailman Cargo Unification

### PR DESCRIPTION
**Pull request by heresthemailitneverfails**
# About The Pull Request
Makes the Mailman job independent from its Gimmick role

## Why It's Good For The Game
So you can play Mailman for round start
More easy to get antag as Mailman

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog

:cl:
tweak: makes mailman a normal role
/:cl:

### CODE REVIEW

**Comment by Blueturbo47** [comment]
you need a changelog, it would probably be 

:cl:
tweak: makes mailmain a normal role
/:cl:

**Comment by heresthemailitneverfails** [comment]
done it

File: monkestation/code/modules/jobs/job_types/mailman.dm [hunk 1]
```diff
@@ -1,16 +1,15 @@
-/datum/job/gimmick/mailman
+/datum/job/mailman
 	title = "Mailman"
 	flag = MAILMAN
 	department_head = list("Head of Personnel")
 	department_flag = CIVILIAN
 	faction = "Station"
-	outfit = /datum/outfit/job/gimmick/mailman
+	outfit = /datum/outfit/job/mailman
```
Comment (lmenvs) [inline monkestation/code/modules/jobs/job_types/mailman.dm:7]: You will need to update the outfit datum below as well if you are removing gimmick from its path.

Comment (heresthemailitneverfails) [reply monkestation/code/modules/jobs/job_types/mailman.dm:7]: okay, where do i find the "/datum" part?

Comment (heresthemailitneverfails) [reply monkestation/code/modules/jobs/job_types/mailman.dm:7]: okay i think i did it, check if i did it right

Comment (heresthemailitneverfails) [reply monkestation/code/modules/jobs/job_types/mailman.dm:7]: okay i know how to do it i just dont know how to add something to the PR

File: code/modules/jobs/jobs.dm [hunk 2]
```diff
@@ -55,7 +55,6 @@ GLOBAL_LIST_INIT(gimmick_positions, list(
 	"Debtor",
 	"Psychiatrist",
 	"VIP",
```
Comment (lmenvs) [inline code/modules/jobs/jobs.dm:57]: remove the comma after VIP and add the two ending parentheses here as well

File: monkestation/code/modules/jobs/job_types/mailman.dm (hunk 1 above)
Comment (lmenvs) [reply monkestation/code/modules/jobs/job_types/mailman.dm:7]: Additional needs on this file:

- Setting the job display_color, check cargo tech or shaft miner for color

- you will need to set the display_order variable, which will also need to be defined.  check cargo tech or shaft miner for this variable, and update the job defines file accordingly.

- rename the file to mailman.dm

**Review by lmenvs — CHANGES_REQUESTED** [review]
Additional changes to files:

-Jobs define needs to be updated to include the JOB_DISPLAY_ORDER_MAILMAN.  These show up in their numerical order on the job prefs screen so I'd just shove it under Shaft Miner.  You have to renumber everything below it as well.

-Mail goodies file needs to be updated to remove gimmick from the mailman datum (not fixing this actually makes gimmick show up twice on job prefs)

-the .dme will need to reflect your file rename


I am gonna go out on a limb here and say you're new to coding.  I encourage you to check out the guide to getting set up with VSC if you aren't already, and make sure to test your code before PRing it! I'd be happy to help you with any questions you have.

File: monkestation/code/modules/jobs/job_types/mailman.dm (hunk 1 above)
Comment (heresthemailitneverfails) [reply monkestation/code/modules/jobs/job_types/mailman.dm:7]: does that look good?

Comment (lmenvs) [reply monkestation/code/modules/jobs/job_types/mailman.dm:7]: this file is fine now.  Please make the changes to code\__DEFINES\jobs.dm and monkestation\code\modules\jobs\mail_goodies\job_mail.dm

Comment (heresthemailitneverfails) [reply monkestation/code/modules/jobs/job_types/mailman.dm:7]: weird i thought i had fixed that

Comment (heresthemailitneverfails) [reply monkestation/code/modules/jobs/job_types/mailman.dm:7]: okay im having serious problems on how to place changes on the same PR and branch, what am i doing wrong?

**Comment by heresthemailitneverfails** [comment]
hey i think the PR ended up too messy... is it cool if i close it and retry it?

**Pull request closed without merging**